x-pack/plugins/ingest_manager/server/saved_objects/index.ts

```diff
@@ -54,6 +54,8 @@ const savedObjectTypes: { [key: string]: SavedObjectsType } = {
         type: { type: 'keyword' },
         active: { type: 'boolean' },
         enrolled_at: { type: 'date' },
+        unenrolled_at: { type: 'date' },
+        unenrollement_started_at: { type: 'date' },
```

Comment:
small typo, should be `unenrollment_started_at` (in other places for this field too)